rebasehelper/application.py

```diff
@@ -544,8 +544,6 @@ def build_source_packages(self):
                 build_dict.update(builder.build(spec, results_dir, **build_dict))
                 build_dict = self._sanitize_build_dict(build_dict)
                 results_store.set_build_data(version, build_dict)
-            except RebaseHelperError:
-                raise
```

Comment:
Actually, removing this will lead to discarding the original exception object and raising a new `build failed with unknown reason` error.

Reply:
I think we need to whitelist the `try-except-raise` warning, which means resolving issue #514 first.

Reply:
And I think it would be best to make it a part of this PR.
I'm proposing the following changes to `tox.ini`:

```diff
diff --git a/tox.ini b/tox.ini
index 3e622fd..f96b83e 100644
--- a/tox.ini
+++ b/tox.ini
@@ -26,18 +26,22 @@ passenv=
     PYTEST_ADDOPTS
     REQUESTS_CA_BUNDLE
 commands=
-    {[lint]commands}
     py.test --verbose --color=yes \
             {posargs:--cov rebasehelper --cov-report html --cov-report term}
 deps=
     -rtest-requirements.txt
-    -rlint-requirements.txt
 
 [testenv:py2]
 basepython=python2
 
 [testenv:py3]
 basepython=python3
+commands=
+    {[lint]commands}
+    {[testenv]commands}
+deps=
+    {[lint]deps}
+    {[testenv]deps}
 
 [lint]
 skip_install=True
@@ -53,11 +57,11 @@ deps=
 [testenv:lint-py2]
 basepython=python2
 skip_install={[lint]skip_install}
-commands = {[lint]commands} --disable=import-error
-deps = {[lint]deps}
+commands={[lint]commands} --disable=import-error
+deps={[lint]deps}
 
 [testenv:lint-py3]
 basepython=python3
 skip_install={[lint]skip_install}
-commands = {[lint]commands} --disable=import-error
-deps = {[lint]deps}
+commands={[lint]commands} --disable=import-error
+deps={[lint]deps}
```